dnf-behave-tests/features/dnf-transaction-exception.feature

```diff
@@ -0,0 +1,15 @@
+@no_installroot
+Feature: Test exception when dnf tries to do a transaction resolved but not valid anymore
+
+Background: setup repository
+  Given I use repository "dnf-transaction"
+    And I configure dnf with
+        | key          | value                                 |
+        | pluginpath   | /dnf-plugins |
+    And I do not disable plugins
+
+
+Scenario: Dnf fails to update a plugin if it does not exist after resolving the transaction
+  Given I successfully execute dnf with args "install wilee"
+   When I execute dnf with args "remove update wilee"
```

Comment:
So IIUC this `wilee` package installs a plugin and then the test actually works on the plugin package, which seems to be an unnecessary "inception" complication that may be a bit confusing? Better perhaps to have one package with the plugin and another package that is being installed/upgraded.

Also, the command is `remove update wilee`, AFAICS it's a mistake, this calls `remove` on two packages: `update` and `wilee`.

Reply:
regarding `remove`, `update`. I was distracted my bad. anyway the exception should trigger both with `remove` and `update` command